libs/auth/src/certs.ts

```diff
@@ -29,6 +29,15 @@ const VX_CUSTOM_CERT_FIELD = {
   ELECTION_HASH: `${VX_IANA_ENTERPRISE_OID}.4`,
 } as const;
 
+/**
+ * Standard X.509 cert fields, common across all VotingWorks certs
+ */
+const STANDARD_CERT_FIELDS = [
+  'C=US', // Country
+  'ST=CA', // State
+  'O=VotingWorks', // Organization
+] as const;
```

Comment:
TBD what these fields will actually end up being. We also may have to move these somewhere more global since machine certs will also need these fields.